### ISSUE TITLE
Value Descriptions

### ISSUE THREAD

**Issue by KamasamaK** [foundeo/cfdocs#1069]
It seems that there is a need for proper `param.values` descriptions. These are often being placed in the `param.description` (or sometimes the top-level `description`) as unordered lists, but there is a slight redundancy and clear disconnect from `param.values`. I've seen an effort to move away from that, but I think it still has value and we should have a solution for that.

I've encountered the same situation with VS Code configuration, and they have a good solution that would not break compatibility. They have an `enum` array and `enumDescriptions` array that are supposed to match in number of elements and correspond the enum value to its description based on position in the respective arrays. [Here](https://github.com/KamasamaK/vscode-cflint/blob/bd1c7f7a8929c72d74a0ebbd52dc14517c161b74/package.json#L131) is a simple example from my **vscode-cflint** extension.

To match the existing naming scheme, I propose we add either `valueDescriptions` or `valuesDescriptions`. Whichever name is fine, but it would be an array the same as `enumDescriptions`.

**Comment by pfreitag** [comment]
Love it - lets do it.

**Comment by shaedrich** [comment]
So if I understand correctly that looks technically a bit like `callback_params`, doesn't it?

**Comment by pfreitag** [comment]
To implement this there would be 3 tasks:

* Update the view to display them
* Update the readme
* Update the JSON schema

**Comment by pfreitag** [comment]
@shaedrich Yeah it would be similar to `callback_params`, so I guess the question is, is it useful to have other metadata as well (such as engine details)? Possibly it could be useful to denote that engine x doesn't support this `action` value, or requires version y

**Comment by shaedrich** [comment]
That would be the level which is missing in #525, right?

**Comment by KamasamaK** [comment]
So, would you instead like a `valuesMetadata` property which is an array of objects?
